### ISSUE TITLE
Add option to export each collection to a separate file

### ISSUE THREAD

**Issue by robintw** [robintw/AutoZotBib#2]
Add option to export each collection to a specific file, named after that collection (or configurable). Option as to whether to also export all items too. Could also allow settings for which collections are to be included in 'all'.

See https://github.com/robintw/AutoZotBib/commit/f0f0178ac5540d9e15ea5305ac60894f28b34e45#commitcomment-1140765 for original suggestion.

**Comment by joonro** [comment]
I was just about to open an issue about this. :) Would be great if this is implemented.

**Comment by robintw** [comment]
This is definitely on the todo list - and I'm about to start using Zotero plus AutoZotBib a lot more, so it is likely to creep up to the top of my todo list fairly shortly. I can't guarantee exactly when I'll be able to do it - but hopefully within the next couple of weeks.

**Comment by joonro** [comment]
Sounds good. Thank you!

**Comment by btel** [comment]
Has there been any progress in this area? I would actually prefer to have an option to select a collection (or set of collections) to export.

Thanks for the great tool!

**Comment by Cadair** [comment]
@btel I agree it would be awesome to be able to export a specific collection to a specific file.

**Comment by fgregg** [comment]
:+1:
